config/workload_all.xml

```diff
@@ -26,8 +26,11 @@
     	</transactiontype>
    	</transactiontypes>
 
-    <terminals>2</terminals>
     <scalefactor>2</scalefactor>
+	<!-- The number of terminals can be a max of 10 per warehouse -->
+    <terminals>20</terminals>

```

Comment:
> Because we are waiting so long and each terminal is not doing much: e.g 30 seconds on average for NewOrder, we would ideally want to increase parallelism to increase performance.
The maximum allowed according to specs is 10 per warehouse and we used that.

Also there are no unwanted transaction failures due to this, so we are fine here.

Reply:
Got it, thanks! Guessing the default number of warehouses is 2, so this looks good.